src/build.py

```diff
@@ -864,26 +873,25 @@ def LLVM():
   Mkdir(LLVM_OUT_DIR)
   cc_env = BuildEnv(LLVM_OUT_DIR, bin_subdir=True)
   build_dylib = 'ON' if not IsWindows() else 'OFF'
-  command = [PREBUILT_CMAKE_BIN, '-G', 'Ninja', LLVM_SRC_DIR,
-             '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON',
-             '-DLLVM_BUILD_TESTS=ON',
-             '-DCMAKE_BUILD_TYPE=Release',
-             '-DCMAKE_INSTALL_PREFIX=' + INSTALL_DIR,
-             '-DLLVM_INCLUDE_EXAMPLES=OFF',
-             '-DCOMPILER_RT_BUILD_XRAY=OFF',
-             '-DCOMPILER_RT_INCLUDE_TESTS=OFF',
-             '-DCOMPILER_RT_ENABLE_IOS=OFF',
-             '-DLLVM_BUILD_LLVM_DYLIB=%s' % build_dylib,
-             '-DLLVM_LINK_LLVM_DYLIB=%s' % build_dylib,
-             # Our mac bot's toolchain's ld64 is too old for trunk libLTO.
-             '-DLLVM_TOOL_LTO_BUILD=OFF',
-             '-DLLVM_INSTALL_TOOLCHAIN_ONLY=ON',
-             '-DLLVM_ENABLE_ASSERTIONS=ON',
-             '-DLLVM_TARGETS_TO_BUILD=X86;WebAssembly']
-
-  command.extend(OverrideCMakeCompiler())
+  command = CMakeCommand([
+      LLVM_SRC_DIR,
+      '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON',
```

Comment:
Might as well move this into `CMakeCommand` too actually. If it's useful anywhere (which I'm not 100% sure it is, but :woman_shrugging:) then it's useful in all/any of the CMake uses.